docs/cloud-infrastructure-security/index.md

```diff
@@ -0,0 +1,47 @@
+---
+slug: /cloud-infrastructure-security
+title: Cloud Infrastructure Security
+tags: [cloud infrastructure Security, security]
```

Comment:
I might also add the acronym tag here in case customers search for it this way.

```suggestion
tags: [cloud infrastructure Security, cis, security]
```

Reply:
Good idea, @kimsauce.